jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/state/CodeModernizerSessionState.kt

```diff
@@ -29,23 +29,24 @@ class CodeModernizerSessionState {
     var transformationSummary: TransformationSummary? = null
     var currentJobId: JobId? = null
 
-    private fun getJobItemId(sessionContext: CodeModernizerSessionContext) = Path(sessionContext.configurationFile.path).toAbsolutePath().toString()
-    fun putJobHistory(sessionContext: CodeModernizerSessionContext, status: String, startedAt: Instant = Instant.now()) {
-        val id = getJobItemId(sessionContext)
+    private fun getJobModuleName(sessionContext: CodeModernizerSessionContext) = Path(sessionContext.configurationFile.path).toAbsolutePath().toString()
+    fun putJobHistory(sessionContext: CodeModernizerSessionContext, status: String, jobId: String = "", startedAt: Instant = Instant.now()) {
+        val module = getJobModuleName(sessionContext)
```

Comment:
nit: `module` should be called something like `moduleName` since it reads like its an object. But its a string.